### ISSUE TITLE
evaluation on DAVIS2016

### ISSUE THREAD

**Issue by Avocado5818** [z-x-yang/CFBI#46]
Hello, I am using pre-trained weights (trained on YT+DAVIS) name=ResNet101-CFBI-DAVIS, J&F=81.9, the result eval on davis17 is correct. But the eval score on davis16 is lower than the result of the paper(I use the same weight eval on davis16), so I want to know which weight or how to run the eval result on davis16? 
Thanks!!

**Comment by z-x-yang** [comment]
The currently available checkpoints are reproduced by using the released code. Note that the code doesn't fix any random seed during the training process. Thus, the results could be slightly different from the results reported in our paper.

**Comment by Avocado5818** [comment]
Thank you for your reply, so I have to train on YT first and then fine-tuning to find the best score on DAVIS2016, right?
Thanks!!

**Comment by z-x-yang** [comment]
Yeah.  Or you could fine-tune from our checkpoint pretrained on YT.

**Comment by Avocado5818** [comment]
Ok, I got it, thank you very much.

**Comment by pytho12** [comment]
Hello, I have trained your code, but there is a 1.2% difference in the accuracy of both DAVIS16 and DAVIS17 datasets. Are there any tips that can help improve the code accuracy?
